matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/message/LocationInfo.kt

```diff
@@ -22,7 +22,7 @@ import com.squareup.moshi.JsonClass
 @JsonClass(generateAdapter = true)
 data class LocationInfo(
         /**
-         * Required. RFC5870 formatted geo uri 'geo:latitude,longitude;uncertainty' like 'geo:40.05,29.24;30' representing this location.
+         * Required. RFC5870 formatted geo uri 'geo:latitude,longitude;uncertainty' like 'geo:40.05,29.24;u=30' representing this location.
```

Comment:
I would update also the pseudo code to

```suggestion
         * Required. RFC5870 formatted geo uri 'geo:latitude,longitude;u=uncertainty' like 'geo:40.05,29.24;u=30' representing this location.
```

Reply:
Thanks, it seems I missed some comment updates. Fixed in [5fa3b3f](https://github.com/vector-im/element-android/pull/6376/commits/5fa3b3f16862b1f4b43e6f017faa372b5fd29ed7)